packages/cli/package.json

```diff
@@ -10,6 +10,7 @@
     "access": "public"
   },
   "dependencies": {
+    "@apidevtools/swagger-parser": "^10.0.2",
```

Comment:
Any reason/benefit for the switch? AFAIK, both are actively maintained as aliases.